src/hotspot/share/runtime/arguments.cpp

```diff
@@ -568,6 +568,7 @@ static SpecialFlag const special_jvm_flags[] = {
   { "UsePSAdaptiveSurvivorSizePolicy",                  JDK_Version::undefined(), JDK_Version::jdk(26), JDK_Version::jdk(27) },
 
   { "PretenureSizeThreshold",       JDK_Version::undefined(), JDK_Version::jdk(26), JDK_Version::jdk(27) },
+  { "HeapMaximumCompactionInterval",JDK_Version::undefined(), JDK_Version::jdk(26), JDK_Version::jdk(27) },
```

Comment:
Maybe add a space?
```suggestion
  { "HeapMaximumCompactionInterval", JDK_Version::undefined(), JDK_Version::jdk(26), JDK_Version::jdk(27) },
```

Reply:
That did cross my mind. However, I went for the current version as I feel aligning with previous line (identical obsolete/expire version) is more important.

WDYT?

Reply:
What I would do in this case is to make L571 look nice and adapt the alignment of L570 accordingly, but I'm not sure if there's anything about this in the style guide. Feel free to ignore the comment